app/poros/quote_search.rb

```diff
@@ -0,0 +1,9 @@
+class QuoteSearch
+  attr_reader :num_results,
+              :quotes
+
+  def initialize(quote_data)
+    @num_results = quote_data[:totalCount]
+    @quotes = quote_data[:results].map { |quote| Quote.new(quote) }
```

Comment:
Nice job instantiating another PORO here to get specific info for a quote. You could also do this at the `QuoteFacade` level.